src/main/java/subway/auth/service/AuthService.java

```diff
@@ -0,0 +1,56 @@
+package subway.auth.service;
+
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.stereotype.Service;
+import org.springframework.util.ObjectUtils;
+import subway.auth.dto.TokenRequest;
+import subway.auth.dto.TokenResponse;
+import subway.auth.infrastructure.JwtTokenProvider;
+import subway.common.exception.InvalidLoginException;
+import subway.member.dao.MemberDao;
+import subway.member.domain.Member;
+
+@Service
+public class AuthService {
+    private JwtTokenProvider jwtTokenProvider;
+    private MemberDao memberDao;
+
+    public AuthService(JwtTokenProvider jwtTokenProvider, MemberDao memberDao) {
+        this.jwtTokenProvider = jwtTokenProvider;
+        this.memberDao = memberDao;
+    }
+
+    private boolean checkValidLogin(StringBuilder errorMessageBuilder, String email, String password) {
+        try {
+            if (!memberDao.findByEmail(email)
+                    .getPassword()
+                    .equals(password)) {
+                errorMessageBuilder.append(InvalidLoginException.EMAIL_PASSWORD_MISMATCH);
+                return false;
+            }
+        } catch (EmptyResultDataAccessException erdae) {
+            errorMessageBuilder.append(InvalidLoginException.EMAIL_NOT_EXIST);
+            return false;
+        }
+        return true;
+    }
+
+    public TokenResponse createToken(TokenRequest tokenRequest) {
+        StringBuilder errorMessageBuilder = new StringBuilder();
+        if (tokenRequest.isEmpty()
+                || !checkValidLogin(errorMessageBuilder, tokenRequest.getEmail(), tokenRequest.getPassword())) {
+            throw new InvalidLoginException(errorMessageBuilder.toString());
+        }
+        String accessToken = jwtTokenProvider.createToken(tokenRequest.getEmail());
+        return TokenResponse.of(accessToken);
+    }
```

Comment:
checkValidLogin에서 리턴타입을 없앤다면 여기도 이렇게 변경해볼 수는 있어요. (참고만 해주세요.) 
```suggestion
    public TokenResponse createToken(TokenRequest tokenRequest) {
        if (tokenRequest.isEmpty()) {
            throw new IllegalArgumentException("tokenRequest invalid");
        }
        checkValidLogin(tokenRequest.getEmail(), tokenRequest.getPassword());
        String accessToken = jwtTokenProvider.createToken(tokenRequest.getEmail());
        return TokenResponse.of(accessToken);
    }

Reply:
넵 알겠습니다!